.github/workflows/deploy.yml

```diff
@@ -29,6 +29,7 @@ jobs:
     - run: |
         git config --global user.name 'AmbNum Bot'
         git config --global user.email 'ambnum.bot@disinfo.quaidorsay.fr'
+    - run: npm run validate:schema
```

Comment:
> Why do we test in the `deploy` workflow? 🤨

Reply:
Because this workflow is executed after the merge on master, and it is to re-test after an eventual conflicts resolution before deploying.